localization/localization.zhCN.lua

```diff
@@ -88,6 +88,8 @@ L["Do you want to resume recurring award (%s) %d EP/%s?"] = "你想重新开始
 L["EP Reason"] = "EP 理由"
 L["EP/GP are reset"] = "EP/GP 已被重置"
 L["EP/GP/PR announce medium"] = "EP/GP/PR 通告媒介"
+L["EP/GP/PR announce text"] = "EP/GP/PR announce text"
+L["Default:\n${char} EP:${ep} GP:${gp} PR:${pr}\n\nAdditional Variables:\n${bid}"] = "Default:\n${char} EP:${ep} GP:${gp} PR:${pr}\n\nAdditional Variables:\n${bid}"
```

Comment:
You don't have to modify this file. Just leave these two items blank.

Reply:
I was just following the pattern from the other text there - I wasn't sure if it was a "reminder" to translate later or whether to just let the i18n file take care of itself.  Sounds good to me!